clang/lib/AST/MicrosoftMangle.cpp

```diff
@@ -2986,6 +2989,9 @@ void MicrosoftCXXNameMangler::mangleCallingConvention(CallingConv CC) {
     case CC_Swift: Out << 'S'; break;
```

Comment:
The default case here should be a proper compiler error, not an unreachable, since there's no good way to prevent arbitrary calling conventions from reaching the mangler.

If you search this file, you can find examples of "cannot yet mangle". Please copy that error handling pattern and use it in the default case for this switch to improve handling of cases like this in the future.

Reply:
Done